lamindb/models/artifact.py

```diff
@@ -2225,7 +2232,9 @@ def cache(self, is_run_input: bool | None = None, **kwargs) -> Path:
 
         Args:
             is_run_input: Whether to track this artifact as run input.
-            **kwargs: Keyword arguments for synchronization.
+            mute: Whether to print the progress of caching.
+            **kwargs: Keyword arguments for internal caching functions.
```

Comment:
Please shorten the explanation and move it into a shared docstring variable. You can just say "Internal arguments for debugging." or something like this? That's what you explained why it's needed, right?

Reply:
done